docs/hooks/use-navigate.md

```diff
@@ -24,31 +47,47 @@ function useLogoutTimer() {
 }
 ```
 
-## Type Declaration
+The `navigate` function has two signatures:
 
-```tsx
-declare function useNavigate(): NavigateFunction;
+- Either pass a `To` value (same type as `<Link to>`) with an optional second `options` argument (similar to the props you can pass to [`<Link>`][link]), or
+- Pass the delta you want to go in the history stack. For example, `navigate(-1)` is equivalent to hitting the back button
 
-interface NavigateFunction {
-  (
-    to: To,
-    options?: {
-      replace?: boolean;
-      state?: any;
-      relative?: RelativeRoutingType;
-    }
-  ): void;
-  (delta: number): void;
-}
-```
+## `options.replace`
 
-The `navigate` function has two signatures:
+Specifying `replace: true` will cause the navigation will replace the current entry in the history stack instead of adding a new one.
+
+## `options.state`
 
-- Either pass a `To` value (same type as `<Link to>`) with an optional second `{ replace, state }` arg or
-- Pass the delta you want to go in the history stack. For example, `navigate(-1)` is equivalent to hitting the back button.
+You may include an optional state value in to store in [history state][history-state]
 
-If using `replace: true`, the navigation will replace the current entry in the history stack instead of adding a new one.
+## `options.preventScrollReset`
+
+When using the [`<ScrollRestoration>`][scrollrestoration] component, you can disable resetting the scroll to the top of the page via `options.preventScrollReset`
+
+## `options.relative`
+
+By default, navigation is relative to the route hierarchy, so `..` will go up one `Route` level. Occasionally, you may find that you have matching URL patterns that do not make sense to be nested, and you'd prefer to use relative _path_ routing. You can opt into this behavior with `relative: true`:
```

Comment:
This should read something like

You can opt into this behavior with `relative: "route" | "path"`